### PR TITLE
Update styles.scss

### PR DESCRIPTION
**Pull request by microdou**
Fix to following warning:
Failed prop type: Required prop `theme.notificationsSystem.className` was not specified in `NotificationsSystem`

### CODE REVIEW

**Comment by sarneeh** [comment]
I think there is no need for inserting all the stuff that You did right there. It could be just:
```css
.notifications-system {
    /*I'm needed here*/
}
```

**Review by jsardev — CHANGES_REQUESTED** [review]
Review below :)

File: src/styles/styles.scss [hunk 1]
```diff
@@ -23,7 +23,10 @@
 }
 
 // NotificationSystem component
-.notifications-system {
+:local .notifications-system {
```
Comment (jsardev) [inline src/styles/styles.scss:26]: How about just:

```
.notifications-system {
    /*I'm needed here*/
}
```

**Review by jsardev — CHANGES_REQUESTED** [review]

File: src/styles/styles.scss (hunk 1 above)
Comment (microdou) [reply src/styles/styles.scss:26]: I was just following `reapop-theme-wybo/blob/master/src/styles/styles.scss` :)

Comment (jsardev) [reply src/styles/styles.scss:26]: Hmm... I'm not sure it's a good choice. Wybo theme could have some different font styles than bootstrap. Correct me if i'm wrong :)

**Comment by LouisBarranqueiro** [comment]
This is no more required. `NotificationSystem.classNames` is now optional - just update reapop module -  https://github.com/LouisBarranqueiro/reapop/releases/tag/v0.6.1

**Comment by jsardev** [comment]
Thanks! :)

**Pull request closed without merging**